src/scripts/components/DropdownWrapper.jsx

```diff
@@ -35,7 +35,7 @@ class DropdownWrapper extends React.Component {
         const $input = $dropdown.querySelector('input[type="text"]');
 
         // If dropdown contains input (for filtering), focus on that
-        if ($input) {
+        if ($input && !this.props.inputBlur) {
           $input.focus();
```

Comment:
@alexgsummer So, whenever I pass inputBlur to dropdown, the focus property is set only when the value of inputBlur is false. In cases when I am not passing inputBlur props, the behaviour of the inputs remains unaffected.